internal/config/config.go

```diff
@@ -126,6 +136,11 @@ func read() Config {
 			config.RyukPrivileged = ryukPrivilegedEnv == "true"
 		}
 
+		ryukAddress := os.Getenv("TESTCONTAINERS_RYUK_ADDRESS")
+		if ryukAddress != "" {
```

Comment:
nit: we could fold this to be a little more idiomatic
```suggestion
		if ryukAddress := os.Getenv("TESTCONTAINERS_RYUK_ADDRESS"); ryukAddress != "" {
```